climpred/utils.py

```diff
@@ -58,8 +64,9 @@ def get_comparison_function(comparison, dict_):
     if isinstance(comparison, types.FunctionType):
         return comparison
     else:
-        is_in_dict(comparison, dict_, 'comparison')
-        return dict_[comparison]
+        is_in_list(comparison, list_, 'comparison')
+        comparison = METRIC_ALIASES.get(comparison, comparison)
```

Comment:
I don't think this line should be here. This is comparisons, not metrics, and there are no comparison aliases.